recipes/skaware/skalibs.inc

```diff
@@ -0,0 +1,66 @@
+DESCRIPTION = "skalibs is a package centralizing the free software / open source C development files used for building all software at skarnet.org"
+LICENSE = "ISC"
+
+SRC_URI = "http://skarnet.org/software/${PN}/${PN}-${PV}.tar.gz"
+
+inherit c make
+
+endianness = "${@['big', 'little'][d.get('HOST_ENDIAN') == 'l']}"
+
+SRC_URI += "file://sysdeps file://sysdeps.h"
+do_configure[prefuncs] += "do_configure_sysdeps"
+do_configure_sysdeps() {
+	rm -rf ${B}/sysdeps
+	mkdir -p ${B}/sysdeps
+	echo "" > ${B}/sysdeps/rt.lib
+	echo "" > ${B}/sysdeps/socket.lib
+	echo "" > ${B}/sysdeps/sysclock.lib
+	echo "" > ${B}/sysdeps/tainnow.lib
+	echo "" > ${B}/sysdeps/util.lib
+	echo "${HOST_ARCH}" > ${B}/sysdeps/target
```

Comment:
don't we need to make it incompatible with other archs, i'm aware you get a build failure, but making it incompatible signals it much earlier at least